style.css

```diff
@@ -19,25 +20,29 @@ header a {
   align-items: center;
 }
 
+.about-me.headline {
+  background-color: white;
+  background-image: none;
+}
+
 .headline {
   display: flex;
   flex-direction: column;
-  height: 100vh;
   padding: 50px 20px;
   background-image: url(images/header-shapes\ mobile.svg);
   background-repeat: no-repeat;
```

Comment:
- [ ] The background image is not set correctly this is the original 👇
![or](https://user-images.githubusercontent.com/61562626/129418546-f08dbb4c-7467-48f8-b123-2c625865f280.jpeg)

- And this is yours 👇, you see the bg image is not fitting in the background, please fix it
![you](https://user-images.githubusercontent.com/61562626/129418580-6dd89ca3-2fd2-4106-9499-04347dc80477.jpeg)